voltdbclient/distributor.go

```diff
@@ -297,6 +316,9 @@ func (c *Conn) loop(disconnected []*nodeConn, hostIDToConnection *map[int]*nodeC
 
 func (c *Conn) submit(pi *procedureInvocation) (int, error) {
 	nc := c.availableConn()
+	if nc.isClosed() {
+		nc = c.availableConn()
+	}
```

Comment:
What's the purpose of retrying get a node connection again here?

Reply:
Nothing really, this just slipped.